stand_moves:
  - both_arms/step1
  - both_arms/step2
  # - both_arms/step3
  # - both_arms/step4
  # - both_arms/step5
  # - both_arms/step6
  # - both_arms/step7

  - both_arms/step9
  - both_arms/step10
  - both_arms/step11
  - both_arms/step10
  - both_arms/step11
  - both_arms/step12
both_arms:
  step1: [0.0, -1.6, 1.5, 0.0, 0.4, 0.0, 0.0, -1.6, 1.5, 0.0, 0.4, 0.0]
  step2: [0.0, 0.0, 1.0, 0.0, 0.570, 0.0, 0.0, 0.0, 1.0, 0.0, 0.570, 0.0]
  step3: [-0.509, 1.210, 0.022, -0.611, 1.570, -0.162,-0.509, 1.210, 0.022, -0.611, 1.570, -0.162]
  step4: [-1.140, 1.384, 0.119, -1.134, 1.570, -0.542, -1.140, 1.384, 0.119, -1.134, 1.570, -0.542]
  step5: [-0.429, 1.155, -0.126, -0.856, 1.500, -0.162, -0.429, 1.155, -0.126, -0.856, 1.500, -0.162]
  step6: [0.836, -0.077, 0.109, 0.056, 1.471, -0.162, 0.836, -0.077, 0.109, 0.056, 1.471, -0.162]
  step7: [1.389, -0.072, 0.118, 0.405, 1.468, -0.099, 1.389, -0.072, 0.118, 0.405, 1.468, -0.099]
  # step8: [0, -1.57, -1.57, 0, 0, 0, -0.5, 1, -0.5, 0, 0, 0.0]
  step9: [0, -1.57, -1.57, 0, 0, 0, -0.5, 1, -1, 0, -0.5, 1.57]
  step10: [0, -1.57, -1.57, 0, 0, 0, -0.5, 1, -0.5, 0, 0, 1.57]
  step11: [0, -1.57, -1.57, 0, 0, 0, -0.5, 1, -1, 0, -0.5, 1.57]
  step12: [0, -1.57, -1.57, 0, 0, 0, -0.5, 1, -0.5, 0, 0, 1.57]
